README.md

```diff
@@ -34,20 +42,75 @@ verification-methods:
     - "/cert1.pem"
     - "https://www.example.com/cert2.pem" 
 ```
+The following attribute MUST be set:
+- `issuer`:  Issuer of DID document, which is the DID itself.
+- `verification-methods` List of public keys used as verification methods in DID document to be generated. scs-did-creator sets JWK as verification method. JWK is formatted according to [RFC 7517](https://datatracker.ietf.org/doc/html/rfc7517#section-4)) as verification method only. s At least one verification method MUST be set:
+  - `key`: Absolute file path to private key file. Using this setting adds JWK as verification method expressed by parameters `n` and `e`, only
+  - `x509`: Either path or url to X.509 certificate chain. Generator uses public key being referred by first certificate as verification method. Specifying certificate as file path adds [`x5c`](), using an URL, adds [`x5u`](https://datatracker.ietf.org/doc/html/rfc7517#section-4.6) parameter to JWK format.
+
```

Comment:
```suggestion
**Note**: To produce a DID document compatible with the Compliance API of the Gaia-X Digital Clearing House (GXDCH), specify a single `x509s` entry using a publicly accessible URL pointing to a valid certificate chain and omit any `keys` entries.

```

I think this is the most crucial non-obvious fact that is still missing from the documentation, which led to #4.